lib/helpers.js

```diff
@@ -128,7 +128,10 @@ var fetchResource = function (resourceUrl, cb) {
     return cb(false);
   }
 
-  moduleForProtocol(resourceUrl).get(resourceUrl, function (response) {
+  moduleForProtocol(resourceUrl).get({
+    hostname: resourceUrl,
+    headers: { 'Origin': 'https://srihash.org/' },
```

Comment:
I guess we should file a follow-up bug if we end up merging this with the hard-coded string.